app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
```

Comment:
I wanted to avoid that kind of notation using the `canI` filter, meaning not passing an objects with nested fields, just simple:
- resource without apiGroup: `ng-if="'buildconfigs' |  canI : 'create'"`
- resource with apiGroup `ng-if="'extensions/horizontalpodautoscalers' | canI 'create'"`

 Also searching in the normalized rules is instant since all keys are uniq based on the `apiGroup/resource` format. Quick searching in the rules was the main motivation behind the rules normalization.
I kinda see why the slash delimiter is not the best way, but we are not checking if the given resource is a sub-resource anyway with the `canI` filter.